pixart.js

```diff
@@ -1 +1,24 @@
+var form = document.querySelector("#form")
+var brush = document.querySelector(".brush")
+var input = document.querySelector("#color-field")
+var button = document.querySelector("#set-color");
 
+form.addEventListener('submit', function(event){
+  event.preventDefault();
+  brush.style.backgroundColor = input.value;
+});
+
+form.addEventListener('onkeydown', function(event){
+  event.preventDefault();
+  brush.style.backgroundColor = input.value;
+});
+
```

Comment:
"submit" handles both "click" and pressing enter. "onkeydown" will trigger everytime a character is entered in the `#form` we really only want the event to fire once the user is done typing the color they want.